### ISSUE TITLE
[Feature Suggestion] Anchor links between journal entries

### ISSUE THREAD

**Issue by jobyburk** [ironmonk88/monks-enhanced-journal#266]
The inclusion of anchor links within a journal is a huge step, which I am now using, so kudos for that! However, I wonder if it would be possible to add functionality for anchor links between pages (journal entries). In layman's terms: click on the link on one page, it brings you to the anchor on a complete different page in the same MEJ tab.

Perhaps it already has this functionality and I'm just not aware of the syntax to make it work. If it does, let me know!

**Comment by emphaticrain** [comment]
I would also love to learn the correct syntax for using the anchor feature. Thanks!
